### DISCUSSION TITLE
Is there a way to get the native 'event videos' into home assistant like on the eufy app?

### DISCUSSION THREAD

**Discussion by natta193** [fuatakgun/eufy_security#1250] [Q&A]
I want to be able to see the past event videos. Such as like at 9am if someone was at the door I want to see the video of that displayed as a card. Cheers.

**Comment by cunode** [comment]
Seeing videos from past events should also allow to download them. With this I could store the event videos on my own cloud, off the device (which could be destroyed by a bad person).
